pkg/relay.go

```diff
@@ -123,7 +124,7 @@ func (rs *DefaultRelay) RegisterValidator(ctx context.Context, payload []types.S
 }
 
 func (rs *DefaultRelay) processValidator(ctx context.Context, payload []types.SignedValidatorRegistration, state State) error {
-	logger := rs.Log().WithField("method", "RegisterValidator")
+	logger := rs.Log().WithField("method", "ProcessValidator")
```

Comment:
I dont think we need to rename this. Having it under registerValdiator lets us associate it with other logs that come from the higher up function. log body messages should be sufficient to delineate